### PR TITLE
matsuo first commit

### PR DESCRIPTION
**Pull request by matsuo-y**
matsuo first commit

### CODE REVIEW

**Comment by opst-fujimoto** [comment]
OK

File: README.md [hunk 1]
```diff
@@ -14,6 +14,30 @@ new branch
...
+ ( '-')
+ ( '-' )
+ ('-' )
+ (-' )
+ (' )
+ ( )
+ (o' )
+ ('o' )
+ ( 'o' )
+ ( 'o')
```
Comment (opst-fujimoto) [inline README.md]: この行はいらない

**Comment by opst-adachi** [comment]
LGTM

**Comment by opst-fujimoto** [comment]
[LGTM](http://lgtm.in/i/guVMDuWsn)

**Comment by opst-adachi** [comment]
[LGTM](http://lgtm.in/i/MkQGaMFNp)

**Comment by matsuo-y** [comment]
[LGTM](http://lgtm.in/i/gEgt5KjBn)

**Pull request merged**